includes/transformer/class-comment.php

```diff
@@ -30,6 +30,10 @@ class Comment extends Base {
 	 * @return int The User-ID of the WordPress Comment
 	 */
 	public function get_wp_user_id() {
+		if ( is_single_user() && \user_can( $this->wp_object->user_id, 'activitypub' ) ) {
+			// Single user sites are federated with the Blog User.
+			return Users::BLOG_USER_ID;
+		}
```

Comment:
This should not be needed, because it is handled by the Dispatcher, the same way it is handled for posts: https://github.com/Automattic/wordpress-activitypub/blob/master/includes/class-activity-dispatcher.php#L46

Reply:
Oh, love to remove code! 👍🏻